webapp/src/components/NonCompliantCard/index.js

```diff
@@ -106,7 +103,7 @@ const NonCompliantCard = ({ producer, stats }) => {
         <Typography variant="overline">{t('dailyRewards')}</Typography>
         <RowInfo
           title={`${t('rewards')} (USD)`}
-          value={`$${formatWithThousandSeparator(
+          value={`$ ${formatWithThousandSeparator(
```

Comment:
I think we can remove the space between the dollar sign and number to maintain cosistency with other views 
It also makes the text shprter
<img width="181" alt="image" src="https://user-images.githubusercontent.com/5632966/217945588-f8054fb8-b076-4d37-a5ee-143a95e04eb2.png">

<img width="68" alt="image" src="https://user-images.githubusercontent.com/5632966/217945639-e906281e-92a3-45ce-ae75-9ae66be443cd.png">


<img width="58" alt="image" src="https://user-images.githubusercontent.com/5632966/217945619-aab387c4-99fb-4450-828c-94c2d2da4480.png">